### DISCUSSION TITLE
Jakob feuern?

### DISCUSSION THREAD

**Discussion by N0ah-S** [Gold-Squadron/XY-Fleet#29] [Polls]
> [!WARNING]
> Falls Jakob nicht gefeuert wird, könnte er vielleicht anfangen Sachen zu commiten.

**Comment by N0ah-S** [comment]
@Lucaauem

**Reply by N0ah-S** [reply]
@any1-source

**Comment by Lucaauem** [comment]
Warum steht ,,nein" überhaupt zur Auswahl?

**Reply by N0ah-S** [reply]
sry
